common/test/acceptance/tests/studio/test_studio_course_info.py

```diff
@@ -0,0 +1,156 @@
+"""
+Acceptance Tests for Course Information
+"""
+from common.test.acceptance.pages.studio.course_info import CourseUpdatesPage
+
+from base_studio_test import StudioCourseTest
+from ...pages.studio.auto_auth import AutoAuthPage
+from ...pages.studio.index import DashboardPage
+
+
+class UsersCanAddUpdatesTest(StudioCourseTest):
+    def setUp(self, is_staff=False, test_xss=True):
+        super(UsersCanAddUpdatesTest, self).setUp()
+        self.auth_page = AutoAuthPage(self.browser, staff=True)
+        self.dashboard_page = DashboardPage(self.browser)
+
+        self.course_name = self.course_info['display_name']
+        self.course_org = self.course_info['org']
```

Comment:
This is only used a couple of lines below-- no need to define a class variable (same for course_number and course_run).